src/styles.css

```diff
@@ -54,26 +48,33 @@ body {
 /* NAV BAR */
 nav {
   display: flex;
-  justify-content: flex-end;
-  align-items: flex-start;
-  width: 100%;
+  justify-content: space-between;
+}
+
+nav > h1 {
+  text-align: center;
+  margin-left: 25px;
+  cursor: pointer;
 }
 
 .nav-child {
   display: flex;
-  justify-content: space-around;
```

Comment:
justify-content was aligning the children elements of nav-child evenly. With this change, the nav has nav-child elements in flexbox but the nav-child elements don't have centered children (they're left-aligned) which could cause problem if something else were to change (I can not imagine the exact situation).
Could we centre the elements inside all nav-child?
I know it looks fine to the user right now but it just feels like a loose end for any future changes.

Reply:
sure i'll add this back then